agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -73,15 +76,39 @@ public String toString() {
     /**
      * @return a Comment as a HTML div with proper formatting to be displayed
      */
-    private String htmlFormat() {
+    private String htmlFormat(String languageCode) {
       // gets time in local time zone (default: US ET)
       LocalDateTime localTime =
           new LocalDateTime(timestamp.getTime(), DateTimeZone.forID("US/Eastern"));
       DateTimeFormatter formatter = DateTimeFormat.forPattern("h:mm a M/dd/yy");
 
       return "<div class='comment-div'>"
           + "<p class='date'>" + formatter.print(localTime) + "</p>"
-          + "<p><b>" + name + " (" + email + "):</b> <br><br>" + text + "</p></div>";
+          + "<p><b>" + name + " (<a href='mailto:" + email + "'>" + email + "</a>):</b></p>"
+          + "<p class='comment-text' lang=" + languageCode + ">" + getTranslatedText(languageCode) + "</p></div>";
+    }
+
+    /**
+     * @return the text field translated to the language of a given language code (e.g. "EN" for English)
+     * [Makes a network call to Translation API so method could be slow]
```

Comment:
In examples I've seen, generic descriptions goes above the tags in JavaDoc. Do you have examples where descriptions go below the tags?

Reply:
That's a great point -- I will move it up above. I think I just accidentally typed it under. Thank you!

Reply:
Done